Makefile

```diff
@@ -3,3 +3,6 @@ PWD              ?= $(shell pwd)
 
 verify: 
 	cat ./Dockerfile
+
+test-e2e:
+	./test/e2e.sh
```

Comment:
The [Google Shell style guide](https://google.github.io/styleguide/shellguide.html#s2.1-file-extensions) prefers removing the `.sh` extension for executables. If we do not have contradicting guideles, how about doing that here?

Reply:
Done. Thanks.